WebDriverAgentLib/Categories/XCElementSnapshot+FBHelpers.m

```diff
@@ -128,6 +132,28 @@ - (XCElementSnapshot *)fb_parentCellSnapshot
     }
     return targetCellSnapshot;
 }
+
+- (CGRect) fb_visibleFrameWithFallback
```

Comment:
please remove the space after closing bracket to align with the common source style